### PR TITLE
Used Rebrandly API to shorten link and updated UI

### PR DESCRIPTION
**Pull request by has12zen**
## Description
Used Rebrandly API to shorten The URL of the Current Page and updated the HTML for better UI.

## Related Issue
 #7 

## Proposed Changes:
   1.  Change 1 Adding a way to show the current page URL as QR code
   1.  Change 2
   

## How Has This Been Tested?
 *Manual testing

# Screenshots
**Updated Screenshot**

Screenshot (42)

Screenshot (43)

Screenshot (44)

Screenshot (45)
.

### CODE REVIEW

File: app/scripts/popup.js [hunk 1]
```diff
@@ -1,22 +1,51 @@
...
+        responseField.innerHTML = `<p>Your shortened url is: </p><p> ${res.shortUrl} </p>`
+    }
+}
+// Some page elements
+const inputField = document.querySelector('#input')
+const shortenButton = document.querySelector('#shorten')
+const responseField = document.querySelector('#responseField')
+const shortenUrl = (urlToShorten) => {
+    const data = JSON.stringify({ destination: urlToShorten })
+    const xhr = new XMLHttpRequest()
```
Comment (maximilianschmelzer) [inline app/scripts/popup.js:38]: We could replace this with modern fetch. So we have promises and could use current `Headers`/`Request`/`Response` interfaces. Also looks more up to date than the multiple `xhr.xyz()` 😄 
```
fetch(apiUrl, {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
    apikey
  },
  body: JSON.stringify(data)
}).then((res) => {
  // wait for res async ...
})
// normal code progresses ...
```
Or is there any specific reason to use the old xml method? 🙂

**Review by ghost — COMMENTED** [review]
.

**Comment by has12zen** [comment]
Thanks, I'll try to do the changes.

File: app/scripts/popup.js (hunk 1 above)
Comment (has12zen) [inline app/scripts/popup.js:38]: no there was no particular reason.

**Pull request closed without merging**